pycbc/results/templates/orange.html

```diff
@@ -100,11 +100,14 @@
 
         <!-- title and subtitle -->
         <div class="row" style="padding-top:10px">
-            <div class="col-md-6">
-                <p class="text-left"><b>{{analysis_title}}</b></p>
+            <div class="col-md-4">
```

Comment:
Although this works, it used to be the case that Bootstrap scaffolding default rows were defined in blocks of 12 (.e.g 6 + 6 as it was previously). https://getbootstrap.com/2.3.2/scaffolding.html

I would suggest altering this so that it equals 12 again.

Reply:
I've added in an empty div so that it still adds up to 12